src/Plugin/Linkit/Substitution/Canonical.php

```diff
@@ -69,7 +69,7 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function getUrl(EntityInterface $entity) {
     /** @var \Drupal\Core\GeneratedUrl $url */
-    $url = parent::getUrl($entity);
+    $url = parent::getUrl($entity)->toString(TRUE);
```

Comment:
this has been changed to tackle this issue - https://digital-vic.atlassian.net/browse/SRM-1167
Recently linkit module did a new release and it changed the getUrl function return from `return $entity->toUrl('canonical')->toString(TRUE);` to `return $entity->toUrl('canonical');`
It causes issues for our custom plugin as it returns the url object instead of string. So to fix that, converting it here in the custom plugin.